src/Main.cc

```diff
@@ -495,6 +495,14 @@ static void showSplash()
   logger->report("OpenROAD {} {}",
                  ord::OpenRoad::getVersion(),
                  ord::OpenRoad::getGitDescribe());
+  logger->report("Features included (+) or not (-):\n"
+                "{} {} {} {}",
+                strcmp(ord::OpenRoad::getChartsCompileOption(), "ON") == 0 ? "+Charts" 
+                                                                           : "-Charts",
```

Comment:
move the common portion to the format string eg

"{}Charts", ord::OpenRoad::getChartsCompileOption() ? '+' : '-'